README.md

```diff
@@ -1,69 +1,88 @@
-Apache SkyWalking
```

Comment:
don;t remove any reference to original README including any other references. Create another README name it was README-CloudScale and add reference to main file

Reply:
okay !